### ISSUE TITLE
importing 'newrelic' doesn't work

### ISSUE THREAD

**Issue by stoikerty** [stoikerty/dev-toolkit#58]
`import 'newrelic';` results in the new relic config file `projectRoot/newrelic.js` being returned instead of the new relic package (which would itself read the config file).

**Comment by stoikerty** [comment]
This issue comes from the way that `babel-plugin-module-resolver` was being configured in `babel-preset-dev-toolkit`.
```js
...
    [
      require.resolve('babel-plugin-module-resolver'),
      {
        // This will allow anything inside the project root directory to be imported
        // useful for using `import settings from 'src/client/settings';` but problematic
        // in the case of newrelic which has uses a file with the same name as the package
        root: [process.cwd()],
      },
    ],
...
```

**Comment by stoikerty** [comment]
fixed with #57
